core/callback.go

```diff
@@ -30,4 +30,7 @@ type RestGenCallback interface {
 // GrpcGenCallback is currently a subset of RestGenCallback so is defined separately for convenience
 type GrpcGenCallback interface {
```

Comment:
Also, please create a `GenCallback` that `RestGenCallback` and `GrpcGenCallback` both implement.

Reply:
ok